package.json

```diff
@@ -6,6 +6,7 @@
   "repository": "https://github.com/BrightspaceUI/testing.git",
   "scripts": {
     "lint": "eslint . --ext .js",
+    "start": "web-dev-server --root-dir ./.vdiff --open ./report/",
```

Comment:
When we wrap the runner, can we expose something like this so it doesn't need to be added to every `package.json` for the consumers? Or maybe we run this at the end of a test run 🤔?

Reply:
I was thinking the same thing but forgot to mention it -- yes! We either do it automatically for visual-diff runs that have more than 1 failure (but not in CI), or add a `--open-report` flag or something (or both).